engines/rbd.c

```diff
@@ -29,6 +33,7 @@ struct rbd_data {
 	rbd_image_t image;
 	struct io_u **aio_events;
 	struct io_u **sort_events;
+    int fd; /* add for poll */
```

Comment:
Thanks. I will take care coding style next time.

One suggestion, how about adding a shell to check the coding style in fio package? If you agree, I could give help, draft a "check_format.sh", and let you review.